.changelog/unreleased/features/3297-mempool-interface-senders.md

```diff
@@ -0,0 +1,2 @@
+- `[mempool]` Add new `GetSenders` method to `Mempool` interface, and `Senders` method to `Entry`
```

Comment:
This has to be duplicated under breaking change too.

Reply:
I moved the file to breaking-changes.